x/mint/expected_keepers.go

```diff
@@ -1,9 +1,14 @@
 package mint
 
-import sdk "github.com/cosmos/cosmos-sdk/types"
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	stake "github.com/cosmos/cosmos-sdk/x/stake"
+)
 
 // expected stake keeper
 type StakeKeeper interface {
+	GetPool(ctx sdk.Context) stake.Pool
+	SetPool(ctx sdk.Context, pool stake.Pool)
```

Comment:
we should just use nah use the function `InflateSupply` (already here) instead

Reply:
ah of course, missed that